extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanClientProducer.java

```diff
@@ -167,12 +167,34 @@ private ConfigurationBuilder builderFromProperties(Properties properties) {
             builder.marshaller((Marshaller) marshallerInstance);
         }
 
-        // Override serverList property value at runtime if such configuration exists
         if (infinispanClientRuntimeConfig != null) {
+            // Override serverList property value at runtime if such configuration exists
             Optional<String> runtimeServerList = infinispanClientRuntimeConfig.serverList;
```

Comment:
Sorry to ask this, but can we change this one to be the same format (ie. using ifPresent lambda) so the code is all consistent.

Reply:
updated